tests/test_pumptrap_calibration.py

```diff
@@ -0,0 +1,274 @@
+import os
+import glob
+import numpy as np
+from corgidrp.mocks import generate_mock_pump_trap_data
+from corgidrp.detector import imaging_area_geometry
+from corgidrp.data import Dataset
+from corgidrp.l1_to_l2a import prescan_biassub
+from corgidrp.l2a_to_l2b import em_gain_division
+from corgidrp.pump_trap_calibration import tpump_analysis, tau_temp
+
+
+
+# Adjust the system's limit of open files. We need to load 2000 files at once. 
+# some systems don't like that. 
+import resource
```

Comment:
do you need a `def teardown_module():` at the end that sets the system open file limit back to its original value?

Reply:
my understanding is that this change is only for the current process and its children, so we shouldn't have to change it back. Is it fine as is then?

Reply:
just checked and this is correct, so all good.